src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,17 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQLDialect</property>
+        <property name="hibernate.connection.url">jdbc:mysql://localhost:3306/hibernate_relations</property>
+        <property name="hibernate.connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="hibernate.connection.username">root</property>
+        <property name="hibernate.connection.password">1234567890</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class = "mate.academy.model.Movie"/>
+        <mapping class = "mate.academy.model.MovieSession"/>
+        <mapping class = "mate.academy.model.CinemaHall"/>
+        <mapping class = "mate.academy.model.User"/>
+
+    </session-factory>
+</hibernate-configuration>
```

Comment:
try to keep an empty line at the end of each file to avoid this in GitHub
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/49767049/952a9970-79e8-44d4-aa13-d625458ba43d)